src/loaders.ts

```diff
@@ -380,6 +397,28 @@ export class SourcifyABILoader implements ABILoader {
 
 export class SourcifyABILoaderError extends errors.LoaderError { };
 
+// Contract metadata from Sourcify
```

Comment:
```suggestion
///  Sourcify Contract metadata from API response
```

Just making it a bit more consistent with the other one.